src/fragments/start/getting-started/flutter/nextsteps.mdx

```diff
@@ -10,3 +10,9 @@
 ### References
 
 Find the source code of the final Todo App built in this tutorial [on our GitHub repository](https://github.com/cshfang/amplify-flutter-datastore-starter/tree/main/amplified_todo).
+
+### Escape Hatch
```

Comment:
Do we typically talk about escape hatches on the getting started tutorial? It seems a bit of an advanced use case for jumping right into when the tutorial assumes a very low level knowledge of Amplify. If other platforms do this then we can be consistent and do it but otherwise it may be enough to simply have it listed somewhere on the libraries sidebar?

Reply:
I think it's more like if you're getting started and are familiar with AWS already you may be curious about how to do what you already know. That's also why it's just a blurb which redirects to the larger example.